src/lib.rs

```diff
@@ -1,177 +1,90 @@
+#![allow(dead_code)]
```

Comment:
:) Perfect! Just leave a comment once it's out of WIP.
I like it so far and turning this into a proper crate would be nice.